pages/linux/parted.md

```diff
@@ -0,0 +1,35 @@
+# parted
+
+> A partition manipulation program.
+
+- List partitions on all block devices:
+
+`parted --list`
+
+- Start to manipulate disk partition:
+
+`parted {{/dev/sdX}}`
+
+- Create a new partition table of label-type directly, label-type can be gpt, msdos etc:
+
+`parted --script {{/dev/sdX}} {{mklabel gpt}}`
```

Comment:
Reading the man page, I think `mklabel` is an argument like `--script`. The argument option is `gpt`:
```suggestion
`parted --script {{/dev/sdX}} mklabel {{gpt}}`
```

Reply:
You are right. But `mklabel` can be other command like `resizepart` etc. So I'm not sure whether we should put mklabel in the curly brackets.

Reply:
True, but in this case the `mklabel` is required for the example to make sense @einverne, so I  think @navarroaxel's suggestion would be better :slightly_smiling_face: